ci/start_init_test.sh

```diff
@@ -611,6 +623,60 @@ test_replica_rpc_close() {
 	cleanup
 }
 
+verify_bad_file_descriptor_error() {
+        echo "-----------------Verify_bad_file_descriptor_error-------------"
+        verify_replica_cnt "2" "Two replica count test"
+        if [ "$1" == "When punching holes while preload" ];
+        then
+            run_ios_rand_write
+            docker stop $orig_controller_id
+            sleep 1
+            docker start $orig_controller_id
```

Comment:
preload will not cause any impact.. we need to do IOs and restart controller one more time.. and also need to restart replica not in debug mode

Reply:
ok, sounds good